CPAC/registration/registration.py

```diff
@@ -4352,45 +4035,20 @@ def warp_deriv_mask_to_EPItemplate(wf, cfg, strat_pool, pipe_num, opt=None):
                 "EPI-template"],
      "outputs": ["space-EPItemplate_res-derivative_desc-bold_mask"]}
     '''
-    xfm_prov = strat_pool.get_cpac_provenance(
-        'from-bold_to-EPItemplate_mode-image_xfm')
-    reg_tool = check_prov_for_regtool(xfm_prov)
-
-    num_cpus = cfg.pipeline_setup['system_config'][
-        'max_cores_per_participant']
-
-    num_ants_cores = cfg.pipeline_setup['system_config']['num_ants_threads']
-
-    apply_xfm = apply_transform(f'warp_deriv_mask_to_EPItemplate_{pipe_num}',
-                                reg_tool, time_series=False,
-                                num_cpus=num_cpus,
-                                num_ants_cores=num_ants_cores)
-
-    apply_xfm.inputs.inputspec.interpolation = "NearestNeighbor"
-
-    node, out = strat_pool.get_data("space-bold_desc-brain_mask")
-    wf.connect(node, out, apply_xfm, 'inputspec.input_image')
-
-    node, out = strat_pool.get_data("EPI-template")
-    wf.connect(node, out, apply_xfm, 'inputspec.reference')
```

Comment:
Based on the similar NodeBlock https://github.com/FCP-INDI/C-PAC/blob/9e2d593f55a842e5be686bf369dfa7e3b9751eda/CPAC/registration/registration.py#L4124-L4174 and the overall pattern of these functions, should the `inputspec.reference` here be `EPI-template-deriv` instead of `EPI-template`?

Right now I'm pretty sure this NodeBlock
https://github.com/FCP-INDI/C-PAC/blob/9e2d593f55a842e5be686bf369dfa7e3b9751eda/CPAC/registration/registration.py#L4339-L4385 produces the exact same output with a different resource pool key as this NodeBlock https://github.com/FCP-INDI/C-PAC/blob/9e2d593f55a842e5be686bf369dfa7e3b9751eda/CPAC/registration/registration.py#L4291-L4336

Reply:
@sgiavasis looks like you wrote the original NodeBlock (https://github.com/FCP-INDI/C-PAC/commit/e4ef3c7c2bb2aa9d2652dda1ba50b18e212e7c55). Can you confirm or reject my suspicion here :point_up_2: